### PR TITLE
Add the ability to destroy a story

### PR DESCRIPTION
**Pull request by brunto**
It would be great to enable the deletion of a story.

### CODE REVIEW

**Comment by silva96** [comment]
Looks great! what do you think about replacing the word "destroy" for "delete" ? We as developers are very used to the concept of destroying a record, but users are more used to "delete" their data.

**Comment by brunto** [comment]
You're right, this is done.

File: app/controllers/stories_controller.rb [hunk 1]
```diff
@@ -29,6 +29,12 @@ def create
     end
   end
 
+  def destroy
+    @story = current_user.stories.find(params[:id])
+    @story.destroy
+    redirect_to dashboard_path
```
Comment (silva96) [inline app/controllers/stories_controller.rb]: What do you think of adding a notice so visual feedback appears after destroying?

```suggestion
    redirect_to dashboard_path, notice: I18n.t('story_deleted_successfully')
```

You would need to add the translation keys too.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -8,6 +8,8 @@
           <p><%= story.excerpt %></p>
           <div class="text-right mt-4">
             <%= link_to t('read_story'), story_path(story), class: 'text-indigo-600 hover:text-indigo-900' %>
+            |
+            <%= link_to t('delete_story'), story_path(story), data: { turbo_method: :delete, turbo_confirm: t('confirm_delete_story') }, class: 'text-indigo-600 hover:text-red-900' %>
```
Comment (silva96) [inline app/views/users/show.html.erb]:
```suggestion
            <%= link_to t('delete_story'), story_path(story), data: { turbo_method: :delete, turbo_confirm: t('confirm_delete_story') }, class: 'text-red-500 hover:text-red-900' %>
```

**Pull request merged**